lib/DataStore.js

```diff
@@ -550,6 +550,25 @@ DataStore.prototype.storeIrcClientConfig = function(config) {
     });
 };
 
+DataStore.prototype.getUserFeatures = function(userId) {
+    return this._userStore.getMatrixUser(userId).then((matrixUser) => {
+        if (!matrixUser) {
+            return {};
+        }
+        return matrixUser.get("features") || {};
```

Comment:
This all could be shortened to `return (matrixUser ? matrixUser.get("features") : {}) || {}` if you wanted